### PR TITLE
BZ-1857790 Added note about live migration of VMs using HPP

### PR DESCRIPTION
**Pull request by sjhala-ccs**
BZ-1857790 https://bugzilla.redhat.com/show_bug.cgi?id=1857790

Added an explicit note about live migration of VMs created from common templates and using HPP or another storage class with RWO access mode as well as passthrough features such as SRI-OV and GPU.

Preivew build: https://bz-1857790--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-features-for-storage.html#virt-features-for-storage-matrix_virt-features-for-storage

Tagging @aglitke for SME review
Tagging @rnetser for QE review

Merge/CP to enterprise-4.5 and enterprise-4.6

### CODE REVIEW

**Review by aglitke — APPROVED** [review]

**Comment by rnetser** [comment]
Please add a note that "evictionStrategy: LiveMigrate" should not be configured for these VMs.

File: modules/virt-features-for-storage-matrix.adoc [hunk 1]
```diff
@@ -31,3 +31,10 @@
 1. PVCs must request a ReadWriteMany access mode
 2. PVCs must request a ReadWriteOnce access mode
 --
+
+[NOTE]
+====
+Virtual machines using a storage class with ReadWriteOnce (RWO) access mode or passthrough features such as
```
Comment (ousleyp) [inline modules/virt-features-for-storage-matrix.adoc:37]: s/using/that use

For readability, I'd suggest something like the following:

```
[NOTE]
====
You cannot live migrate virtual machines that use:

* A storage class with ReadWriteOnce (RWO) access mode
* Passthrough features such as SRI-OV and GPU

Do not set the `evictionStrategy` field to `LiveMigrate` for these virtual machines.
====
```

**Review by ousleyp — COMMENTED** [review]
This mostly LGTM, but I have one nit and a suggestion that I think would improve readability. Let me know when it's ready to merge. Thanks, Shikha!

File: modules/virt-features-for-storage-matrix.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply modules/virt-features-for-storage-matrix.adoc:37]: Edited per your suggestion. Thanks.

**Pull request merged**